Source/Widgets/Timeline/Timeline.js

```diff
@@ -232,13 +232,54 @@ Timeline.prototype.destroy = function () {
 };
 
 /**
- * @private
+ * Shows an highlighted area on the timeline
+ *
+ * @param {Object} config
+ * @param {String|Color} config.color The highlight background color. Can be a css string or a Cesium.Color
+ * @param {number} config.height The height of the highlight area
+ * @param {number} config.base The bottom margin between the highlight range and the bottom of the timeline
+ * @param {JulianDate} config.startTime The start time of the highlight range
+ * @param {JulianDate} config.endTime The end time of the highlight range
+ *
+ * @returns {String} The new HighlightRange id
  */
-Timeline.prototype.addHighlightRange = function (color, heightInPx, base) {
-  var newHighlightRange = new TimelineHighlightRange(color, heightInPx, base);
+Timeline.prototype.addHighlightRange = function (config, heightInPx, base) {
+  // TODO::Deprecation part - remove on deprecation
```

Comment:
What's this TODO message for?  You can see how to add a `deprecationWarning` here: https://github.com/CesiumGS/cesium/tree/master/Documentation/Contributors/CodingGuide#deprecation-and-breaking-changes

Reply:
@hpinkos 
How would you suggest to deprecate this?
This is a private function, so it's not really a deprecation.
Would it be better to create `addNewHighlightRange` and deprecate `addHighlightRange`?

Reply:
I've added the deprecation message. In order to cope with the standards, I've created a new method `addNewHighlightRange` and deprecated the old method `addHighlightRange`.
See the tests for details.